src/main/java/dan200/computercraft/shared/peripheral/monitor/ClientMonitor.java

```diff
@@ -33,6 +34,7 @@
     public BlockPos lastRenderPos = null;
 
     public int tboBuffer;
+    public ByteBuffer monitorBuffer;
```

Comment:
Can we call this something like `tboContents`? `monitorBuffer` could refer to _anything_.

Reply:
Sure.